### PR TITLE
CNV:13246 - Info added to poweroff

### PR DESCRIPTION
**Pull request by sabrinajess**
Re: [CNV-13246](https://issues.redhat.com/browse/CNV-13246)
For 4.8 and later

Preview link: https://deploy-preview-35916--osdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog

### CODE REVIEW

**Comment by kbidarkar** [comment]
LGTM

File: modules/virt-defining-a-watchdog.adoc [hunk 1]
```diff
@@ -9,7 +9,7 @@ Define how the watchdog proceeds when the operating system (OS) no longer respon
 
 .Available actions
 [horizontal]
-`poweroff`:: The virtual machine (VM) powers down immediately.
+`poweroff`:: The virtual machine (VM) powers down immediately. If `spec.running` is in use, or `spec.runStrategy` is not set to `manual`, then the VM will reboot.
```
Comment (sjhala-ccs) [inline modules/virt-defining-a-watchdog.adoc:12]: By "in use" do you mean that `spec.running` is `true` or could the attribute be set to either `true` or `false`? That part is not clear.

Comment (sjhala-ccs) [inline modules/virt-defining-a-watchdog.adoc:12]: s/will reboot/reboots

**Review by sjhala-ccs — COMMENTED** [review]
A couple of nits, otherwise LGTM

File: modules/virt-defining-a-watchdog.adoc (hunk 1 above)
Comment (sabrinajess) [reply modules/virt-defining-a-watchdog.adoc:12]: @kbidarkar Please review.

Comment (kbidarkar) [reply modules/virt-defining-a-watchdog.adoc:12]: Would be good to update "in use" to "true".

**Review by kbidarkar — CHANGES_REQUESTED** [review]

**Pull request merged**